backend/audit/intakelib/checks/runners.py

```diff
@@ -0,0 +1,138 @@
+from django.core.exceptions import ValidationError
+import logging
+from audit.fixtures.excel import FORM_SECTIONS
+
+############
+# General checks
+from .check_uei_exists import uei_exists
+from .check_is_a_workbook import is_a_workbook
+from .check_look_for_empty_rows import look_for_empty_rows
+
+############
+# Federal awards checks
+from .check_is_right_workbook import is_right_workbook
+from .check_state_cluster_names import state_cluster_names
+from .check_other_cluster_names import other_cluster_names
+from .check_direct_award_is_not_blank import direct_award_is_not_blank
+from .check_passthrough_name_when_no_direct import passthrough_name_when_no_direct
+from .check_loan_guarantee import loan_guarantee
+from .check_no_major_program_no_type import no_major_program_no_type
+from .check_missing_award_numbers import missing_award_numbers
+from .check_all_unique_award_numbers import all_unique_award_numbers
+from .check_sequential_award_numbers import sequential_award_numbers
+from .check_num_findings_always_present import num_findings_always_present
+from .check_cluster_name_always_present import cluster_name_always_present
+from .check_federal_award_passed_always_present import (
+    federal_award_passed_always_present,
+)
+
+############
+# Audit findings checks
+from .check_no_repeat_findings import no_repeat_findings
+from .check_findings_grid_validation import findings_grid_validation
+
+logger = logging.getLogger(__name__)
+
+general_checks = [is_a_workbook, uei_exists, look_for_empty_rows]
+
+federal_awards_checks = general_checks + [
```

Comment:
Missing checks for subrecipient validation and a check for program validation here. Adding this here just for reference as I've already mentioned it in the original branch